### DISCUSSION TITLE
FontAwesomeIcon icon prop interface doesn't accept string

### DISCUSSION THREAD

**Discussion by Ryan-Maia** [software-mansion/react-native-svg#2060] [Q&A] [answered]
I was trying to use use the component with Global Import using the `library.add()` method. 

This is the way I'm trying to use it: `<FontAwesomeIcon icon="fa-solid fa-bars" />`

But when I pass a string to the icon prop, it show's me this error: `Type '"fa-solid fa-bars"' is not assignable to type 'IconProp'.`

I look up the interface definition for the Props _(node_modules\@fortawesome\react-native-fontawesome\index.d.ts)_ and added a string option to the icon prop like this: 

```ts
export interface Props {
  icon: IconProp | string;
  ...
}
```

But this change is only temporary on node_modules. I don't know if this is a issue or if there is a correct way to do this. Please someone help me out!

**Comment by jakex7** [comment, marked as answer]
Hey!
To save changes on `node_modules` you can use `patch-package` (https://www.npmjs.com/package/patch-package) however, you should consider rising an issue/discussion in https://github.com/FortAwesome/react-native-fontawesome repository as this issue isn't related to `react-native-svg`.
